__tests__/app.test.js

```diff
@@ -0,0 +1,48 @@
+const app = require('../index');
+const request = require('supertest');
+const db = require('../db/connection');
+const express = require('express');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data/index')
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  test('sends GET to endpoint -> responds with object with two properties', () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then((res) => {
+        expect(res.body[0]).toMatchObject({
```

Comment:
I would make sure that I test all of the objects inside the array and not just the index zero! You could make use of a forEach + expect.any just to check that the properties exist and the values are in correct data types. No need to check exact values for now.